include/cache.h

```diff
@@ -51,7 +51,7 @@ class Cache {
 		const std::string& feedurl);
 	void update_rssitem_unread_and_enqueued(RssItem* item,
 		const std::string& feedurl);
-	void cleanup_cache(std::vector<std::shared_ptr<RssFeed>> feeds,
+	int cleanup_cache(std::vector<std::shared_ptr<RssFeed>> feeds,
```

Comment:
It's impossible to have a negative number of unreferenced feeds, so please make the return type unsigned. Also, it'd be slightly easier to port to Rust if this had a fixed size, so please use go with a fixed-width type like `std::uint64_t`.

Also, please add a little doc-comment (starts with `///`) explaining what this function returns.